client/src/components/JurisdictionAdmin/generateSheets.ts

```diff
@@ -795,3 +797,128 @@ export const downloadTallyEntryLoginLinkPrintout = async (
   )
   return doc.output() // returned for test snapshots
 }
+
+const AUDIT_BOARD_SIGN_MAX_CONTEST_NAME_LENGTH = 40
+const AUDIT_BOARD_SIGN_MAX_SURNAME_LENGTH = 20
+const AUDIT_BOARD_SIGN_TITLE_LENGTH_XL_FONT_CUTOFF = 12
+const AUDIT_BOARD_SIGN_TEXT_COLOR = Colors.GRAY3
+const AUDIT_BOARD_SIGN_TITLE_COLORS = [
+  Colors.GREEN1,
+  Colors.VIOLET1,
+  Colors.ORANGE1,
+  Colors.VERMILION1,
+  Colors.BLUE1,
+  Colors.COBALT1,
+  Colors.GOLD1,
+  Colors.SEPIA1,
+  Colors.DARK_GRAY1,
+  Colors.INDIGO1,
```

Comment:
I noticed some of the colors looked pretty similar in the sample (specifically the bluish ones and the orangish ones). How do they look when printed? Are they distinguishable?

I like using the dark variants for high contrast readability - good choice there. Maybe going one variant lighter would help with differentiation?

Reply:
Good points, I'll make some edits there. But, I don't actually have a color printer with me, so I'll post in the group and ask for someone to validate